### PR TITLE
vaccination alerts: fetch vaccination info from Firestore

### PR DESCRIPTION
**Pull request by pnavarrc**
This PR updates the generate-vaccination-alerts script to fetch the vaccination info version from Firebase and move the `getFirebase` function to a common location since it will be reused for vaccine alerts.

### CODE REVIEW

File: scripts/vaccination_alerts/utils.ts [hunk 1]
```diff
@@ -23,14 +24,13 @@ export function getCmsVaccinationInfo(): RegionVaccinePhaseInfoMap {
...
-  return {
-    // '29': {
-    //   emailAlertVersion: 0,
-    // },
-  };
+export async function getFirebaseVaccinationInfo(): Promise<
+  RegionVaccineVersionMap
+> {
+  const db = await getFirestore();
+  const versionDoc = await db.doc('info/vaccinationInfoUpdates').get();
```
Comment (mikelehen) [inline scripts/vaccination_alerts/utils.ts:31]: Oh, hrm...  I didn't realize we'd store this as a single document.  I was imagining we'd have a collection with a single document for each fips code...  But storing it as a single document is probably easier and should be fine as long as it doesn't exceed 1MB, which it probably won't unless we start adding additional per-fips data to it...  So we can do this, at least for now!

**Review by mikelehen — APPROVED** [review]

File: scripts/vaccination_alerts/utils.ts (hunk 1 above)
Comment (pnavarrc) [reply scripts/vaccination_alerts/utils.ts:31]: Let's change it now then so we don't have to worry about that limitation, I will take than into account for the `vaccine-alerts` collection too 👍

Comment (pnavarrc) [reply scripts/vaccination_alerts/utils.ts]: Updated the Firebase structure, let me know if it makes sense!

**Review by mikelehen — APPROVED** [review]
Awesome, yeah, this is what I had in mind.

If we wanted to flatten our hierarchy a bit we could collapse the `info/vaccinationInfoUpdates/locations` collection into just a top-level `vaccinationInfoUpdates` collection. I think `info` as a top-level collection isn't super useful.

Oh, also, it looks like we're mixing naming conventions a bit.  It looks like today we use `-` in our collection names (alerts-subscriptions, shared-component-params, etc.) and use camelCase for field names within documents.  Should we stick with that?

That would mean `vaccinationInfoUpdates` would be `vaccination-info-updates` (or maybe `vaccination-location-info` or something?) and also `emailAlertVersions` would be `email-alert-versions` ?

**Comment by pnavarrc** [comment]
The `info` part was just reusing the collection that we have to store the last snapshot that we used for risk alerts, we can flatten the structure a bit.

**Pull request merged**